indexer/feature_meta.cpp

```diff
@@ -76,6 +76,8 @@ bool Metadata::TypeFromString(string const & k, Metadata::EType & outType)
     outType = Metadata::FMD_DESTINATION;
   else if (k == "destination:ref")
     outType = Metadata::FMD_DESTINATION_REF;
+  else if (k == "junction:ref")
+    outType = Metadata::FMD_JUNCTION_REF;
```

Comment:
Is there any difference in junction:ref and ref tag?
For highway=motorway_junction we already should take refs into (like) name. Can we make the same with junction:ref.

I suspect that we can _merge_  ref and junction:ref

Reply:
The reason why do we have all these tags separately to have an ability to handle incomplete data.
E.g. only JUNCTION_REF exists for link, but we can find calculate DESTINATION_REF from main road (by it's REF).

Reply:
Ah, I understood. junction:ref is used for _outgoing highway way_, not in motorway_junction ..

Reply:
Is it necessary if we should process motorway_junction ?

Reply:
tags for exit:
junction:ref is number of the junction (exit).
destination:ref - ref of next main road. Normally should match it's ref.

Added comment at RoadNameInfo definition definition.

Reply:
Right, currently I don't care about `motorway_junction`.
But of course it would be better to get it, and use it's tag `ref` instead of tag `junction:ref` of link it last tag does not exists.

Reply:
@vng maybe you can guide me how to merge it in proper way? Or just change the code if it's easier than to explain 😄

Reply:
No need to merge. Yes, this tags are used for highway ways, so everything is fine.

Reply:
I just want to clarify.
Some links have `junction:ref` tag, and some have not.
So we can fill missed data by finding appropriate junction point in the beginning of link, and use it's `ref` tag.

Reply:
We can do it in the next iteration.